api/src/main/java/com/datatorrent/api/DAG.java

```diff
@@ -41,11 +43,26 @@
 {
   interface InputPortMeta extends Serializable, PortContext
   {
+    /**
+     * Return port object represented by this InputPortMeta
+     * @return
+     */
+    public Operator.InputPort<?> getPortObject();
```

Comment:
Can we make call the method getPort to make it consistent with what we have for OperatorMeta where it is getOperator